<launch>
	<!-- USBL -->
    <!-- driver 
    <node name="usbl" pkg="evologics_ros" type="evologics_node.py" output="screen">
    <rosparam command="load" file="$(find evologics_ros)/conf/param_usbl.yaml" />
    </node> -->

	<!--acoustic_link -->
	<node pkg="acoustic_link" name="acoustic_link" type="acoustic_link_node" output="screen">
		<rosparam command="load" file="$(find acoustic_link)/conf/topics_usbl.yaml"/>
		<remap from="/im/in" to="/usbl/im/in"/>
		<remap from="/im/out" to="/usbl/im/out"/>
	</node> 

	<!-- Position -->
 	<node pkg="usbl_position" name="usbl_position" type="usbl_position" output="screen"/>
 	<!-- Plot 
 	<node pkg="usbl_position" name="plot_position" type="plot_position_node"/> -->
 	 <!--rviz 
  	<node pkg="rviz" type="rviz" name="rviz" args="-d $(find usbl_position)/config/plotting.rviz" /> -->



  	<!-- new node: GPS driver -->
</launch>
